docs/src/main/resources/content/_using/replication-installing.adoc

```diff
@@ -27,13 +27,12 @@ For this scenario first copy the `replication.policy` file into the `<ddf-home>/
 
 
 
-==== Installing Via Kar Command
-
-On an already installed system run: `kar:install mvn:replication.distributions.features/replication/${project.version}/kar`
-
 ==== Installing Via Hot-Deploy Directory
 
-Copy the replication kar file into `<ddf-home>/deploy`
+. From the command line uninstall replication by running `feature:uninstall replication`
+. From the command line run `feature:remove-repo replication-x.y.z`, where x.y.z is the current version of replication.
+  * To find the version, run `feature:list | grep -i replication`.
+. Copy the new replication kar file into `<ddf-home>/deploy`
```

Comment:
Isn't this more like updating via Hot-Deploy Directory? And since this is in the "Installation When Replication _*Not*_ Included In Distribution" They shouldn't need to uninstall replication and remove the repo right? Also I just noticed the word "is" is missing from line 24 and there's a typo on line 26 "ths".

Reply:
Nice catch, I'll add another section.